scripts/o3de/o3de/repo_properties.py

```diff
@@ -17,13 +17,142 @@
 import logging
 import hashlib
 import shutil
+import requests
+import sys
+import msvcrt
```

Comment:
It looks like this is a windows only module - what happens if you use the [`getpass.getpass`](https://docs.python.org/3/library/getpass.html#module-getpass) functionaltiy instead?